src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,113 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_ARRAY_SIZE = 10;
+    private Object[] arrayList = new Object[DEFAULT_ARRAY_SIZE];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

Reply:
question: why need to do that. Is it "saving memory" thing or what?

Reply:
no, it's just redundant because int's default value  is 0